tests/performance/test_data_analysis/statistical_data_analyzer.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""
+Library of the Statistical Data Analyzer Module
+
+This library contains the classes and methods necessary to perform statistical analysis and calculations
+on different data sets.
+
+Classes:
+    - DataLoader: class that obtains and loads all the necessary information in their respective variables.
+    - StatisticalComparator: class that performs a comparison between the statistics of two data sets.
+    - StatisticalTests: class that performs statistical tests to detect significant differences in the data.
+"""
+
+import os
+import yaml
+import pandas as pd
+
+from prettytable import PrettyTable
+from collections.abc import Callable
+from typing import Any, Tuple, List, Dict
+from scipy.stats import ttest_ind, levene, f_oneway
```

Comment:
```suggestion
import os
from collections.abc import Callable
from typing import Any, Dict, List, Tuple

import pandas as pd
import yaml
from prettytable import PrettyTable
from scipy.stats import f_oneway, levene, ttest_ind
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3